src/components/Comments.jsx

```diff
@@ -1,5 +1,47 @@
-const Comments = () => {
-  return <h2>Comments</h2>;
+import { useEffect, useState } from "react";
+import { getComments } from "./api";
+
+const Comments = (props) => {
+  const [isLoading, setIsLoading] = useState(true);
+  const [comments, setComments] = useState([]);
+  const article_id = props.article_id;
+
+  useEffect(() => {
+    console.log(article_id);
+    setIsLoading(true);
+    getComments(article_id).then((comments) => {
+      setComments(comments);
+      setIsLoading(false);
+    });
+  }, [article_id]);
+
+  return (
```

Comment:
Could you include something to indicate if there are no comments for the article?